iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/DataBrokerProtectionIOSManager.swift

```diff
@@ -0,0 +1,180 @@
+//
+//  DataBrokerProtectionIOSManager.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Combine
+import Common
+import BrowserServicesKit
+import PixelKit
+import os.log
+import Subscription
+import UserNotifications
+import DataBrokerProtectionCore
+import WebKit
+
+public class DefaultOperationEventsHandler: EventMapping<OperationEvent> {
+
+    public init() {
+        super.init { event, _, _, _ in
+            switch event {
+            default:
+                print("event happened")
+            }
+        }
+    }
+
+    override init(mapping: @escaping EventMapping<OperationEvent>.Mapping) {
+        fatalError("Use init()")
```

Comment:
We might also be able to mark this with `@available(*, unavailable)` to prevent it from being used at compile time (hope I'm remembering that correctly!)